cranelift/fuzzgen/src/function_generator.rs

```diff
@@ -858,8 +864,8 @@ static OPCODE_SIGNATURES: Lazy<Vec<OpcodeSignature>> = Lazy::new(|| {
                 | (Opcode::ResumableTrapnz, &[I32], &[])
                 | (Opcode::ResumableTrapnz, &[I64], &[])
                 | (Opcode::ResumableTrapnz, &[I128], &[])
+                | (Opcode::Call, &[I32], &[])
```

Comment:
`Call` doesn't take any fixed value parameters, right? I think this exception is only necessary for `CallIndirect`, and it was okay that this wasn't in the exceptions list before.

Reply:
Oops, that's right